README.md

```diff
@@ -12,11 +12,12 @@ Important notes:
 
 ### Using command line
 
-#### Configure and generate build files:
+#### Configure and generate C++ build files:
 
     mkdir -p $NUPIC_CORE/build/scripts
     cd $NUPIC_CORE/build/scripts
     cmake $NUPIC_CORE/src [-DCMAKE_INSTALL_PREFIX=../release]
+    export NUPIC_CORE_RELEASE=$NUPIC_CORE/build/release
```

Comment:
This was somewhat counterintuitive since `nupic` has the user specifying `--nupic-core-dir` on the setup.py command line (also breaking convention).